packages/fiori-generator-shared/src/guided-answers.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Mapping of human readable help topic to implementation specific ids
+ * Help items are nodes under a root tree item. A node may have n additonal nodes.
+ *
+ */
+export const enum HELP_TREE {
+    FIORI_TOOLS = 3046
+}
+
+/**
+ * Individual help topics which are mapped to node ids
+ */
+export const HELP_NODES = {
```

Comment:
Much of the GA code is already existing in `odata-service-inquirer` error-handler.

Reply:
Removed from `odata-service-inquirer`

Reply:
I think the `help-topics` file should also be removed, and the GA icon should go with the help nodes?

Reply:
I did delete `help-topics` in the commit https://github.com/SAP/open-ux-tools/pull/2198/commits/4eff5c9cc3416ef7a76825897f270810a6e5c6af

But it's not getting reflected here for some reason

Reply:
Ah never mind it's because it moved to `packages/fiori-generator-shared/src/guided-answers.ts`

Reply:
The two guided answers files are now in fiori gen shared